docs/en/sql-reference/dictionaries/index.md

```diff
@@ -2240,17 +2238,15 @@ We introduce a type of source called `YAMLRegExpTree` representing the structure
       version: '10'
 ```
 
-The key `regexp` represents the regular expression of a tree node. The name of key is same as the dictionary key. The `name` and `version` is user-defined attributions in the dicitionary. The `versions` (which can be any name that not appear in attributions or the key) indicates the children nodes of this tree.
-
-**Back Reference**
-
-The value of an attribution could contain a back reference which refers to a capture group of the matched regular expression. Reference number ranges from 1 to 9 and writes as `$1` or `\1`.
-
-During the query execution, the back reference in the value will be replaced by the matched capture group.
+This config consists of a list of RegExpTree nodes. Each node has following structure:
 
-**Query**
+- **regexp** means the regular expression of this node.
+- **user defined attributions** is a list of dictionary attributions defined in the dictionary structure. In this case, we have two attributions: `name` and `version`. The first nodes have both attributions. The second node only has `name` attribution, because the `version` is defined in the children nodes.
```

Comment:
The rest of the document talks about "attributes". The term "attribution" occurs nowhere else. Could we just use "attributes"?